content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/volume.md

```diff
@@ -0,0 +1,1751 @@
+
+  <a name="PersistentVolumeClaimVolumeSource"></a>
+  *PersistentVolumeClaimVolumeSource references the user's PVC in the same namespace. This volume finds the bound PV and mounts that volume for the pod. A PersistentVolumeClaimVolumeSource is, essentially, a wrapper around another type of volume that is owned by someone else (the system).*
+
+  - **persistentVolumeClaim.claimName** (string), required
+    claimName is the name of a PersistentVolumeClaim in the same namespace as the pod using this volume. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#persistentvolumeclaims
+
+  - **persistentVolumeClaim.readOnly** (boolean)
+    readOnly Will force the ReadOnly setting in VolumeMounts. Default false.
+-->
+### 暴露的持久卷 {#exposed-persistent-volumes}
+- **persistentVolumeClaim** (PersistentVolumeClaimVolumeSource)
+  
+  persistentVolumeClaimVolumeSource 表示对同一名字空间中 PersistentVolumeClaim 的引用。更多信息：
+  https://kubernetes.io/zh-cn/docs/concepts/storage/persistent-volumes#persistentvolumeclaims
+  
+  <a name="PersistentVolumeClaimVolumeSource"></a>
+  **PersistentVolumeClaimVolumeSource 引用同一名字空间中用户的 PVC。
+  此卷找到绑定的 PV 并为 Pod 挂载这个 PV 卷。
+  PersistentVolumeClaimVolumeSource 本质上是其他人（或系统）拥有的另一类卷的包装类。**
+  
+  - **persistentVolumeClaim.claimName** (string)，必需
+    
+    claimName 是与使用此卷的 Pod 位于同一名字空间中的 PersistentVolumeClaim 的名称。更多信息：
+    https://kubernetes.io/zh-cn/docs/concepts/storage/persistent-volumes#persistentvolumeclaims
+  
+  - **persistentVolumeClaim.readOnly** (boolean)
+    
+    readOnly 将在 VolumeMounts 中强制设置 readOnly。默认为 false。
+
+<!--
+### Projections
+- **configMap** (ConfigMapVolumeSource)
+  configMap represents a configMap that should populate this volume
+
+  <a name="ConfigMapVolumeSource"></a>
+  *Adapts a ConfigMap into a volume.
+  The contents of the target ConfigMap's Data field will be presented in a volume as files using the keys in the Data field as the file names, unless the items element is populated with specific mappings of keys to paths. ConfigMap volumes support ownership management and SELinux relabeling.*
+
+  - **configMap.name** (string)
+    Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+
+  - **configMap.optional** (boolean)
+    optional specify whether the ConfigMap or its keys must be defined
+-->
+### 投射 {#projections}
+- **configMap** (ConfigMapVolumeSource)
+  
+  configMap 表示应填充此卷的 configMap。
+  
+  <a name="ConfigMapVolumeSource"></a>
+  **将 ConfigMap 适配到一个卷中。目标 ConfigMap 的 Data 字段的内容将以文件的形式显示在一个卷中，
+  使用 Data 字段中的键作为文件名，除非 items 元素由键到路径的特定映射进行填充。
+  ConfigMap 卷支持所有权管理和 SELinux 重新打标签。**
+  
+  - **configMap.name** (string)
+    
+    被引用资源的名称。更多信息：
+    https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/names/#names
+  
+  - **configMap.optional** (boolean)
+    
+    optional 指定是否必须定义 ConfigMap 或其键。
+<!--
```

Comment:
Within item list, we have to indent the comment block as well, or the following item would have wrong indention.
```suggestion
  <!--
```